tests/mcs/tso/server_test.go

```diff
@@ -120,6 +120,12 @@ func (suite *tsoServerTestSuite) TestTSOServerStartAndStopNormally() {
 }
 
 func (suite *tsoServerTestSuite) TestTSOServerRegister() {
+	for i := 0; i < 3; i++ {
```

Comment:
Why add a loop?

Reply:
check whether can get a new address when the primary update

Reply:
Why adding a loop can do the check?

Reply:
`checkServerRegister` will add a tso server or resourse_manager server, so we can check whether the primary changed.